k8s/charts/openebs/README.md

```diff
@@ -61,19 +61,24 @@ helm del --purge <openebs-chart-name>
 
 The following tables lists the configurable parameters of the OpenEBS chart and their default values.
```

Comment:
The following table lists the configurable parameters of OpenEBS chart and its default values